plugins/micron/micron-nvme.c

```diff
@@ -260,7 +251,7 @@ static int SetupDebugDataDirectories(char *strSN, char *strFilePath,
 					free(fileLocation);
 					goto exit_status;
 				}
-				strcpy(strTemp, fileLocation);
+				strscpy(strTemp, fileLocation);
```

Comment:
Doesn't `strscpy()` require a 3rd parameter (`count`)?

Reply:
This was an un-intended change, reverted back. Thanks for catching!